config/routes.rb

```diff
@@ -257,6 +257,10 @@
     resources :epic_queues, only: [:index, :destroy]
     resources :epic_queue_records, only: [:index]
 
+    resource :protocol_merge do
+      get :perform_protocol_merge
```

Comment:
Because this route is used to manipulate data, it would be better suited as a `PUT` or `POST` https://www.w3schools.com/tags/ref_httpmethods.asp

Reply:
Good catch.